app/controllers/sessions_controller.rb

```diff
@@ -172,6 +194,16 @@ def setup_mfa_authentication
     session[:mfa_user] = @user.id
   end
 
+  def setup_captcha_verification
+    create_catpcha_user(id: @user.id)
```

Comment:
Per my local testing, captcha is also created when trying to login with unknown email, that means `@user` is nil in here and it crashes.

Reply:
The same seems applied when using wrong password for existing account. `find_user` returns nil -> `@user` is nil.